NEWS.md

```diff
@@ -0,0 +1,303 @@
+# 1.7
+
+<!--
+TODO:
+- reword things
+- thank more ppl?
+- sub/3 fixes?
+- validate module metadata is object fix?
+- fix number lexer to avoid conflict with object indexing (#2254)
+- mention some build system changes?
+-->
+
+Lots of JSON has been queried and transformed since the stable release 5 years ago, so now we're happy to finally do a new stable release!
+
+Since the last stable release many things have happened:
+
+- jq now lives at https://github.com/jqlang
+- New and old maintainers have emerged.
```

Comment:
```suggestion
- A new team of maintainers has been organized.
```

(Optionally including a parenthetical clarifying that some are new and some are old).

Reply:
Changed to @nicowilliams suggestion "New maintainers, admins, and owners have been recruited.". What do you think @nicowilliams?